### PR TITLE
chore: clean up old search components

### PR DESCRIPTION
**Pull request by zchsh**
## 🔗 Relevant links

- [Preview link][preview] 🔎
- [Asana task][task] 🎟️

## 🗒️ What

Cleans up old search component code, deleting the previous implementation of search, since we've now fully switched over to "unified" search (with the `All` tab).

Also removes the `enable_unified_search` config flag, as it is no longer used.

## 🧪 Testing

- [ ] Visit the [preview]
     - Search functionality should continue to work as expected

[task]: https://app.asana.com/0/1202097197789424/1204988413642758/f
[preview]: https://dev-portal-git-zscleanup-old-search-components-hashicorp.vercel.app/

### CODE REVIEW

File: src/components/command-bar/commands/search/unified-search/components/index.tsx [hunk 1]
```diff
@@ -3,4 +3,4 @@
  * SPDX-License-Identifier: MPL-2.0
  */
 
-export * from './dialog-body'
+export { UnifiedSearchCommandBarDialogBody } from './dialog-body'
```
Comment (thiskevinwang) [inline src/components/command-bar/commands/search/unified-search/components/index.tsx:6]: 👏

**Comment by zchsh** [comment]
ℹ️ Build failure looks to be `Packer plugins` related, restarting the build now...

File: config/base.json [hunk 2]
```diff
@@ -38,7 +38,6 @@
 	"flags": {
 		"enable_datadog": true,
 		"enable_io_beta_cta": true,
-		"enable_unified_search": true,
```
Comment (kendallstrautman) [inline config/base.json:41]: 🥳

**Review by kendallstrautman — APPROVED** [review]
Played around on the preview and LGTM! 🧼 💯 🔪

**Pull request merged**